README.md

```diff
@@ -1,91 +1,64 @@
+to read from the Reverse Log Proxy instead, though this option is not recommended because of firehose scalability limits.
 
-#### Creating and Uploading Release
+Log Cache is queried by Cloud Controller for app instance metrics such as CPU usage and memory when retrieving details for applications and 
+by the cf cli directly to retrieve recent logs. It can also be queried using the Log Cache CLI plugin to retrieve system component metrics.
 
-The first step in deploying Log Cache is to create a release. Final releases
-are preferable, however during the development process dev releases are
-useful.
 
-The following commands will create a dev release and upload it to an
-environment named `lite`.
+## How do I configure it?
 
-```
-bosh create-release --force
-bosh -e lite upload-release --rebase
-```
+### Scaling
 
-#### Cloud Foundry
+We recomend aiming for retaining 15 minutes of logs and metrics in log-cache per source. This can be monitored with the `cf log-meta` command
+provided by the Log Cache cli plugin or from the log_cache_cache_period metric reported by Log Cache.
 
-Log Cache deployed within Cloud Foundry reads from the Loggregator system and
-registers with the [GoRouter](https://github.com/cloudfoundry/gorouter) at
-`log-cache.<system-domain>` (e.g. for bosh-lite `log-cache.bosh-lite.com`).
+Log Cache nodes use CPU to process incoming envelopes and evict old envelopes from the cache. If your Log Cache runs out of
+CPU resources but has an adequate retention interval you can move Log Cache to an instance type with more CPU.
 
-As of `cf-deployment` version 3.x, Log Cache is included by default in CF.
+Log Cache nodes use memory to cache envelopes. If your Log Cache retention interval is shorter than you would like but you 
+have not exhausted your CPU you can move Log Cache to an instance type with more memory.
 
-The following commands will deploy Log Cache in CF.
+In either case you can add more Log Cache nodes to increase both CPU and memory at the same time.
 
-```
-bosh update-runtime-config \
-    ~/workspace/bosh-deployment/runtime-configs/dns.yml
-bosh update-cloud-config \
-    ~/workspace/cf-deployment/iaas-support/bosh-lite/cloud-config.yml
-bosh \
-    --environment lite \
-    --deployment cf \
-    deploy ~/workspace/cf-deployment/cf-deployment.yml \
-    --ops-file ~/workspace/cf-deployment/operations/bosh-lite.yml \
-    --ops-file ~/workspace/cf-deployment/operations/use-compiled-releases.yml \
-    -v system_domain=bosh-lite.com
-```
+### Reliability
 
-##### Log Cache UAA Client
-By Default, Log Cache uses the `doppler` client included with `cf-deployment`.
+Log Cache is an in memory cache and as such will drop envelopes when it restarts. Users should not expect 100% availability of
+logs in Log Cache and should plan accordingly. For example, Cloud Controller can function without Log Cache though users are
+informed that Log Cache is unavailable.
```

Comment:
Are users informed that Log Cache is unavailable? At least with the `cf app` command the instance quotas are simply replaced with zeros.

Reply:
kinda, sometimes?